Ghidra/Framework/Project/src/main/java/ghidra/framework/plugintool/PluginInfo.java

```diff
@@ -24,15 +24,14 @@
  * <p>
  * Example:
  * <pre>
- * @PluginInfo(
+ * &#64;PluginInfo(
  * 	status = PluginStatus.RELEASED,
  * 	packageName = CorePluginPackage.NAME,
  * 	category = PluginCategoryNames.COMMON,
  * 	shortDescription = "Short description of plugin",
  * 	description = "Longer description of plugin.",
  * 	servicesProvided = { ServiceInterfaceThisPluginProvides.class }
  * 	servicesRequired = { RequiredServiceInterface1.class, RequiredServiceInterface2.class },
- * 	servicesUsed = { OptionalServiceInterface1.class, OptionalServiceInterface2.class },
```

Comment:
The `@` is escaped to prevent javadoc from interpreting it as a tag.
There no longer appears to be servicesUsed for PluginInfo.

Reply:
Can you keep the @ and use a different technique to preserve it please?